velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -286,15 +289,28 @@ VectorPtr WindowFuzzer::buildKRangeColumn(
     if (decodedVector.isNullAt(j)) {
       bits::setNull(rawNulls, j, true);
     } else {
+      const auto orderByValueAtIdx = decodedVector.valueAt<T>(j);
+      if constexpr (std::is_same_v<T, float>) {
+        if (orderByValueAtIdx == kInfFloat ||
+            orderByValueAtIdx == std::nanf("")) {
```

Comment:
nit: use std::isfinite() and std::isnan(), then you don't need to define kInfFloat and kInfDouble, and can catch nan other than std::nan("") or std::nanf("").